File: pset1/mario/less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    printf("Height: ");
    int height = get_int();

    while (height < 0 || height > 23)
    {
        printf("Height: ");

        if (height < 0 || height > 23)
        {
            scanf ("%d", &height);
        }

        else if (height > 0 && height < 23)

        {
            break;
        }
    }


        char hash = '#';

        for (int i = 0; i < height; i++ )
        {
            printf("%*c", height-i, hash);
            printf("%.*s\n", i+1, "########################");

        }



}

File: pset6/mario/less/mario.py
import cs50

# ask user for height
while True:
    print("Height ", end="")
    height = cs50.get_int()
    if height > 0 and height < 23:
        break
# start counter for debug
# count = 0
# use for loop to print
for x in range (0,(height+1),1):
    # count = count + 1
    # print(count)
    print(" " * (height-x), end="")
    print("#" * x)
    # if x = height:
        # break
